website/src/pages/index.js

```diff
@@ -127,8 +136,29 @@ function Home() {
             </div>
           </section>
         )}
+        {showcase && showcase.apps.length > 0 && (
+          <section className='container'>
+            <div className={clsx('row', styles.showcaseTitleContainer)} >
+              <div className={styles.divider} style={{ marginRight: 10 }} />
+              <h1 className={styles.showcaseTitle}>{`See what you can build\nwith Wave`}</h1>
+              <div className={styles.divider} style={{ marginLeft: 10 }} />
+            </div>
+            <div className={clsx('row', styles.showcaseContainer)}>
+              {showcase.apps.map(({ title, description, images }, idx) =>
+                <div className={styles.showcaseRow} key={`row-${idx}`}>
```

Comment:
nit: This should be enough

```suggestion
                <div className={styles.showcaseRow} key={idx}>
```